src/admin-views/editor/panel/ticket.php

```diff
@@ -189,7 +216,132 @@ class="ticket_field ticket_provider"
+							<?php echo Tribe__View_Helpers::is_24hr_format() ? 'data-format="H:i"' : '' ?>
+							data-step="<?php echo esc_attr( $timepicker_step ); ?>"
+							data-round="<?php echo esc_attr( $timepicker_round ); ?>"
+							value="<?php echo esc_attr( $ticket ? $ticket->start_time : null ); ?>"
+							aria-label="<?php
+							echo esc_html(
+									sprintf(
+											__( '%s start date', 'event-tickets' ),
+											tribe_get_ticket_label_singular( 'input_start_time_aria_label' )
+									)
+							); ?>"
+					/>
+					<span class="helper-text hide-if-js"><?php esc_html_e( 'HH:MM', 'event-tickets' ) ?></span>
+					<span class="dashicons dashicons-editor-help" title="<?php
+					echo esc_html(
+							sprintf(
+									__( 'If you do not set a start sale date, %s will be available immediately.', 'event-tickets' ),
+									tribe_get_ticket_label_plural_lowercase( 'input_start_time_help_text_title' )
+							)
+					); ?>">
+			</span>
+				</div>
+			</div>
+			<div class="input_block">
+				<label class="ticket_form_label ticket_form_left" for="ticket_end_date"><?php esc_html_e( 'End sale:', 'event-tickets' ); ?></label>
+				<div class="ticket_form_right">
+					<input
+							autocomplete="off"
+							type="text"
+							class="tribe-datepicker tribe-field-end_date ticket_field"
+							name="ticket_end_date"
+							id="ticket_end_date"
+							value="<?php echo esc_attr( $ticket ? $end_date : null ); ?>"
+					/>
+					<span class="helper-text hide-if-js"><?php esc_html_e( 'YYYY-MM-DD', 'event-tickets' ) ?></span>
+					<span class="datetime_seperator"> <?php esc_html_e( 'at', 'event-tickets' ); ?> </span>
+					<input
+							autocomplete="off"
+							type="text"
+							class="tribe-timepicker tribe-field-end_time ticket_field"
+							name="ticket_end_time"
+							id="ticket_end_time"
+							<?php echo Tribe__View_Helpers::is_24hr_format() ? 'data-format="H:i"' : '' ?>
+							data-step="<?php echo esc_attr( $timepicker_step ); ?>"
+							data-round="<?php echo esc_attr( $timepicker_round ); ?>"
+							value="<?php echo esc_attr( $ticket ? $ticket->end_time : null ); ?>"
+							aria-label="<?php
+							echo esc_html(
+									sprintf(
+											__( '%s end date', 'event-tickets' ),
+											tribe_get_ticket_label_singular( 'input_end_time_aria_label' )
+									)
+							); ?>"
+					/>
+					<span class="helper-text hide-if-js"><?php esc_html_e( 'HH:MM', 'event-tickets' ) ?></span>
+					<span class="dashicons dashicons-editor-help"
+						<?php if ( class_exists( 'Tribe__Events__Main' ) && Tribe__Events__Main::POSTTYPE === get_post_type( $post_id ) ) : ?>
+							title="<?php
+							echo esc_html(
+									sprintf(
+											__( 'If you do not set an end sale date, %s will be available until the event begins.', 'event-tickets' ),
+											tribe_get_ticket_label_plural_lowercase( 'input_end_time_help_text_title' )
+									)
+							); ?>"
+						<?php else : ?>
+							title="<?php
```

Comment:
:no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: dceefbe358ff3551cbf8b92ceda3830d</sub>